eth-contracts/contracts/DelegateManager.sol

```diff
@@ -445,16 +441,11 @@ contract DelegateManager is InitializableV2 {
         );
 
         // Rewards directly allocated to service provider for their stake
-        // Total active funds for direct deployer reward share
-        /// totalActiveDeployerFunds = totalBalanceInSPFactory.sub(spLockedStake);
-        uint256 spRewardShare = (
-            (totalBalanceInSPFactory.sub(spLockedStake)).mul(totalRewards)
-        ).div(totalActiveFunds);
-
+        uint256 spRewardShare = totalRewards.sub(totalDelegatedStakeIncrease);
```

Comment:
Can we add a small comment here about how doing this shifts any loss of precision here?

Reply:
most definitely